test/evergreen.yml

```diff
@@ -2121,14 +2121,15 @@ tasks:
       - func: "get project"
       - func: "compile wiredtiger"
         vars:
-          posix_configure_flags: --disable-shared --with-builtins=snappy,zlib
+          posix_configure_flags: --enable-strict --disable-shared --with-builtins=snappy,zlib
       - command: shell.exec
         params:
           working_dir: "wiredtiger/build_posix"
           shell: bash
           script: |
             set -o errexit
             set -o verbose
+            PATH=/opt/mongodbtoolchain/v3/bin:$PATH
```

Comment:
There's an alternative way of setting the PATH, by replacing `make` with `${make_command|make}` on line 2137, as the desired PATH setting has been covered by the `make_command` variable setting at the ubuntu1804 build variant level.

Reply:
I changed the rule to use `${make_command|make}` as suggested.